ConsentViewController/Classes/SourcePointClient/MessagesResponse.swift

```diff
@@ -22,13 +22,15 @@ enum MessageCategory: Int, Codable, Defaultable, Equatable {
     case gdpr = 1
     case ccpa = 2
     case ios14 = 4
+    case usnat = 6
```

Comment:
Why 6 though? What happened to 5 ?

Reply:
Good question, let me find that out.

Reply:
@Nevazhnovu id `5` is custom campaign (we might support it in the future) and `3` refers to `adBlock` (probably won't support it).